package.json

```diff
@@ -25,58 +25,61 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^14.3.0",
-    "@angular/cdk": "^14.2.7",
-    "@angular/common": "^14.3.0",
-    "@angular/compiler": "^14.3.0",
-    "@angular/core": "^14.3.0",
-    "@angular/elements": "^14.3.0",
-    "@angular/forms": "^14.3.0",
-    "@angular/localize": "14.3.0",
-    "@angular/material": "^14.2.7",
-    "@angular/platform-browser": "^14.3.0",
-    "@angular/platform-browser-dynamic": "^14.3.0",
-    "@angular/router": "^14.3.0",
-    "@angular/service-worker": "14.3.0",
+    "@angular/animations": "^16.2.12",
+    "@angular/cdk": "^16.2.14",
+    "@angular/common": "^16.2.12",
+    "@angular/compiler": "^16.2.12",
+    "@angular/core": "^16.2.12",
+    "@angular/elements": "^16.2.12",
+    "@angular/forms": "^16.2.12",
+    "@angular/localize": "16.2.12",
+    "@angular/material": "^16.2.14",
+    "@angular/platform-browser": "^16.2.12",
+    "@angular/platform-browser-dynamic": "^16.2.12",
+    "@angular/router": "^16.2.12",
+    "@angular/service-worker": "16.2.12",
     "@apollo/client": "^3.9.6",
     "@danielmoncada/angular-datetime-picker": "^16.1.0",
     "@danielmoncada/angular-datetime-picker-moment-adapter": "^3.0.1",
-    "@ng-bootstrap/ng-bootstrap": "13.1.1",
+    "@ng-bootstrap/ng-bootstrap": "15.0.0",
     "@popperjs/core": "^2.11.8",
     "@rxweb/reactive-form-validators": "^13.0.1",
     "@schematics/angular": "^14.2.13",
-    "@swimlane/ngx-graph": "8.0.2",
-    "angular-svg-icon": "^14.0.0",
-    "angular-svg-icon-preloader": "^4.0.1",
-    "angular2-multiselect-dropdown": "^5.0.4",
+    "@swimlane/ngx-graph": "8.2.0",
+    "angular-svg-icon": "^16.1.0",
+    "angular-svg-icon-preloader": "^6.0.0",
+    "angular2-multiselect-dropdown": "^10.0.0",
     "apollo-angular": "^5.0.2",
-    "autoprefixer": "^10.4.18",
+    "autoprefixer": "^10.4.5",
```

Comment:
Rollback?

Reply:
This is precisely the version that does not give a warning in the console during compilation.